bigchaindb/utils.py

```diff
@@ -185,3 +186,23 @@ def load_node_key(path):
         priv_key = priv_validator['priv_key']['value']
         hex_private_key = key_from_base64(priv_key)
         return key_pair_from_ed25519_key(hex_private_key)
+
+
+def check_tendermint_version(running_tm_ver):
+    """
+    Check Tendermint compatability with BigchainDB server
+
+    :param running_tm_ver: Version number of the connected Tendermint instance
+    :type running_tm_ver: str
+    :return: True/False depending on the compatability with BigchainDB server
+    :rtype: bool
+    """
```

Comment:
A neatpick:
```
if not tm_ver:
    return False

for ver in __tm_supported_versions__:
    if version.parse(ver) == version.parse(tm_ver[0]):
        return True

return False
```
Also, in this version it does not fail if the first match fails but only if it does not match any version in the list.